src/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ target_include_directories(mvg
     ${OPENMVG_INCLUDE_DIRS}
     ${Boost_INCLUDE_DIRS}
     ${OpenCV_INCLUDE_DIRS}
+    ${CCTAG_INCLUDE_DIRS}
+    ${TBB_INCLUDE_DIRS}
```

Comment:
Yes, now it's needed because we use CCTag library directly (to retrieve the ellipse). Before that, the usage of CCTag was hidden behind openMVG.